symplyphysics/laws/thermodynamics/energy_to_melt_from_mass.py

```diff
@@ -0,0 +1,35 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output)
+
+# Description
+## The heat of melting is the amount of heat that must be brought to a solid
+# crystalline substance at constant pressure in order to completely transfer it
+# to a liquid state.
+## Q = k_lambda * m
+## Where:
+## Q = energy, required to melt solid matter
+## k_lambda - specific heat of melting
+## m - mass of solid matter
+
+amount_energy = Symbol("amount_energy", units.energy)
+specific_heat_melting = Symbol("specific_heat_melting", units.energy / units.mass)
+mass_of_matter = Symbol("mass_of_matter", units.mass)
+
+law = Eq(amount_energy, specific_heat_melting * mass_of_matter)
```

Comment:
So this law is exactly the same as https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/thermodynamics/energy_from_combustion.py
We probably should keep it but link somehow to another law.

Reply:
The formulas of these laws are very similar, only the meaning of the specific heat constants changes.

Reply:
There is no perfect way to solve it. I suggest to mention in the description about the similarities between these laws. And also put some comment to hint user to clearly distinguish specific heat of combustion, of melting or any other specific heats.

Reply:
Thank you very much!